File: lua/refactoring/tests/refactor/123/csharp/example/inline_var.expected.cs
namespace Test
{
    class Order
    {
        public int quantity { get; set; }
        public int itemPrice { get; set; }
    }

    class Inline
    {
        double orderCalculation(Order order)
        {
            return order.quantity * order.itemPrice -
                Math.Max(0, order.quantity - 500) * order.itemPrice * 0.05 +
                Math.Min(order.quantity * order.itemPrice * 0.1, 100.0);
        }
    }
}

File: lua/refactoring/tests/refactor/123/go/example/inline_var.expected.go
package main

import "math"

type Order struct {
    quantity  float64
    itemPrice float64
}

func orderCalculation(order Order) float64 {
    
    return order.quantity * order.itemPrice - math.Max(0, order.quantity-500)*order.itemPrice*0.05 + math.Min(order.quantity * order.itemPrice*0.1, 100.0)
}
